src/sonic-frr/patch/0010-VRF-interface-lookup-was-still-done-in-the-default-vrf.patch

```diff
@@ -0,0 +1,45 @@
+From 4247dfee7f6bcbb8b4648edba091d71adbb11ce4 Mon Sep 17 00:00:00 2001
```

Comment:
Do we need still need this patch file, looks like we are not using it

Reply:
As FRR updated to 7.5 version, not needed this patch file at all.